.gitignore

```diff
@@ -15,3 +15,5 @@ prerendered/
 aboutlibrary/content/aboutlibrary.bundle.js
 aboutlibrary/content/*.map
 aboutlibrary/content/*.css
+.eslintcache
+
```

Comment:
extra newline. Probably move up just under `.DS_Store`

Reply:
Extra?  There is exactly 1 newline at the end of the file now.  Before, there were 0.

Reply:
That said, I've made the change you suggest, so it's back at 0.